packages/currency/src/currency-utils.ts

```diff
@@ -27,7 +27,7 @@ export const isValidNearAddress = (address: string, network?: string): boolean =
   switch (network) {
     case 'aurora':
     case 'near':
-      return !!address.match(/\.near$/);
+      return !!address.match(/\.(near|tg)$/);
```

Comment:
You can make the capture-group non-capturing since we don't export it to a variable
```suggestion
      return !!address.match(/\.(?:near|tg)$/);
```